src/nyc_trees/apps/core/templates/core/partials/nav.html

```diff
@@ -36,8 +36,10 @@
     <div class="nav-side-sub">
         <h3>NYC</h3>
         <h6>Copyright 2015, The City of New York</h6>
-        <a href="#">Contact Us</a>
-        <a href="#">Privacy Policy</a>
+        <a title="treescount.help@parks.nyc.gov" href="mailto:treescount.help@parks.nyc.gov">Contact Us</a>
```

Comment:
It's embedded in emails too. Building a context processor and DRYing out the templates is a good idea but a little more than I want to tackle right now.